lib/config/rules/ecmascript-6.js

```diff
@@ -39,6 +39,8 @@ module.exports = {
   'prefer-arrow-callback': ['error', {allowNamedFunctions: true}],
   // Suggest using of const declaration for variables that are never modified after declared
   'prefer-const': 'warn',
+  // Require destructuring from arrays and/or objects <FEEDBACK>
+  'prefer-destructuring': ['error', {array: true, object: true}],
```

Comment:
It seems like we should have the `array` off then; I've turned it off. 

I've also enabled the `enforceForRenamedProperties` option. The [schema](http://eslint.org/docs/rules/prefer-destructuring#options) for that options was different than I expected. I thought it would be alongside the obj containing the `array` and `object` options. `enforceForRenamedProperties` seems to be an option within it's own object. 

See 2f2cbf5